src/Snapshooter.NUnit/Snapshooter.NUnit.csproj

```diff
@@ -14,11 +14,15 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
-    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NUnit" Version="4.0.0" />
```

Comment:
Regarding:
> Removed support for netstandard 2.0 due to the fact that NUnit 4.0 no longer supports any .NET Framework below 4.6.2

we should consider referencing a different version of NUnit depending on the Targetframework. for netstandard we can reference the current version, the rest can get nunit 4

Reply:
Or is it better to reference frameworks directly? There are breaking changes in NUnit 4.0, so we have to differ between these in code also, using directives.

Reply:
@PascalSenn @RohrerF This change is a bit problematic because it breaks test projects that target .NET Framework 4.6.2 and reference Snapshooter and also NUnit 4. NuGet will decide to use NuGet 4 in such cases and break Snapshooter which was compiled against NUnit 3.